types/primitive.yaml

```diff
@@ -90,6 +90,13 @@ Bitvector:
   example: "0x01"
   pattern: "^0x[a-fA-F0-9]{2,}$"
 
+Bytes:
+  type: string
+  format: hex
+  description: "An arbitrary-length sequence of bytes"
```

Comment:
arbitrary-length / unbounded values are not SSZ compatible, you always have to define a maximum bound

Reply:
I'll go ahead and close since https://github.com/ethereum/builder-specs/pull/107 was closed as well.  I guess better wording if we ever want to revisit this would be `"An sequence of bytes with unspecified maximum length"`

Reply:
> "An sequence of bytes with unspecified maximum length"

You have to specify a maximum length, there is no unbounded type in ssz. That being said, it would still be possible for the execution requests list use case as you can just update the max length each hard fork if new request types are added